horizons/util/python/decorators.py

```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```

Comment:
I'm not going to comment here further. I requested changes. If you apply them, I'm happy to merge this.

If you really feel this needs more discussion, even though 2! years ago we already had that discussion and made our (two long term maintainers) point clear, open an issue. But I won't tolerate you sneaking your style changes into unrelated pull-requests.